tests/Console/Commands/SendIntegrationActivationReminderEmailCommandTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Console\Commands;
+
+use App\Console\Commands\SendIntegrationActivationReminderEmailCommand;
+use App\Mails\MailJet\MailjetConfig;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Testing\PendingCommand;
+use Tests\TestCase;
+
+final class SendIntegrationActivationReminderEmailCommandTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function test_feature_flag_is_off(): void
```

Comment:
This method name should be more explicit. Something like `test_it_warns_about_disabled_transactional_mails`